### PR TITLE
It's possible to get default parameter value for internal methods/functions in PHP 8

### PR DESCRIPTION
**Pull request by kukulich**

### CODE REVIEW

File: test/unit/SourceLocator/SourceStubber/ReflectionSourceStubberTest.php [hunk 1]
```diff
@@ -295,11 +295,16 @@ private function assertSameParameterAttributes(
...
     ): void {
-        $parameterName = $original->getDeclaringClass()->getName()
-            . '#' . $originalMethod->getName()
-            . '.' . $original->getName();
+        $methodName    = $original->getDeclaringClass()->getName() . '#' . $originalMethod->getName();
+        $parameterName = $methodName . '.' . $original->getName();
 
         self::assertSame($original->getName(), $stubbed->getName(), $parameterName);
+
+        // PHP sometimes reports default value available but it throws error when trying to get the value
```
Comment (Ocramius) [inline test/unit/SourceLocator/SourceStubber/ReflectionSourceStubberTest.php]: This comment seems to be out of sync with the code: I guess we have more default values declared in stubs than in PHP?

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: test/unit/SourceLocator/SourceStubber/ReflectionSourceStubberTest.php (hunk 1 above)
Comment (kukulich) [reply test/unit/SourceLocator/SourceStubber/ReflectionSourceStubberTest.php]: I've to check it again. I'm not able to understand my yesterday's code...

Comment (kukulich) [reply test/unit/SourceLocator/SourceStubber/ReflectionSourceStubberTest.php]: It looks I fixed something else in the night... Fixed the right place now :)

**Review by Ocramius — APPROVED** [review]
🚢  (yes, will wait for CI)

**Comment by Ocramius** [comment]
This is gonna simplify a lot of downstream usage by a lot 💪

**Pull request merged**